Tests/RabbitMq/ConsumerTest.php

```diff
@@ -16,6 +16,7 @@
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 use Symfony\Contracts\EventDispatcher\EventDispatcherInterface as ContractsEventDispatcherInterface;
+use Symfony\Bridge\PhpUnit\ClockMock;
```

Comment:
Please check the possible issues introduced: https://scrutinizer-ci.com/g/eMAGTechLabs/RabbitMqBundle/inspections/29e62c38-e196-4a29-a242-e22f92f6740c/issues/files/Tests/RabbitMq/ConsumerTest.php?status=new&orderField=path&order=asc&honorSelectedPaths=0